main.go

```diff
@@ -154,7 +156,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
+	err = kongCtx.Run(ctx, client)
+	if err != nil {
+		if k8serrors.IsForbidden(err) {
+			fmt.Println("nctl: Permission denied: are you part of the org?	")
+			
+		}else{
+			kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
```

Comment:
This would run the same command again (we already execute it at https://github.com/ninech/nctl/pull/183/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R159). We should just exit here with the error.